pprof/pprof.go

```diff
@@ -3,12 +3,12 @@ package pprof
 import (
 	"context"
 	"net/http"
+	// #nosec G108
```

Comment:
What security issue was it complaining about here?

Reply:
This was just a reordering of the imports. I believe @wojciechos can give more context on the security issue